runtime_config_linux.go

```diff
@@ -198,6 +198,12 @@ type Network struct {
 	Priorities []InterfacePriority `json:"priorities"`
 }
 
+// IntelRdt for Linux cgroup 'intel_rdt' resource management
+type IntelRdt struct {
+	// L3 cache capacity bitmask (CBM) for container
+	L3Cbm *string `json:"l3Cbm,omitempty"`
```

Comment:
On Tue, Dec 22, 2015 at 11:19:43AM -0800, Vish Kannan wrote:
> > +	// L3 cache capacity bitmask (CBM) for container
> > +	L3Cbm *string `json:"l3Cbm,omitempty"`
> 
> Why is it a string and not an integer?

So folks can use hex [1] (which JSON doesn't support [2])?  That's
probably not a good enough reason, since convenient hand-editing is a
lower priority for us than convenient machine readability.  A rule for
handling traditionally-hex values should go into a best-practices doc
(#273) if/when we decide to do that.

[1]: https://lkml.org/lkml/2015/10/2/74
[2]: http://json.org/